src/settings/patronBlocks/Limits/LimitsForm.js

```diff
@@ -18,28 +19,38 @@ import {
 } from '@folio/stripes/components';
 import stripesFinalForm from '@folio/stripes/final-form';
 
-import { feeFineBalanceId } from '../../../constants';
+import {
+  feeFineBalanceId,
+  recallOverdueId,
+} from '../../../constants';
 
 import css from '../patronBlocks.css';
 
-function validation(value, type) {
+function validation(value, min, max) {
   const numberValue = toNumber(value);
-  const min = 0.01;
-  const max = 9999.99;
 
   if (numberValue < min || numberValue > max) {
-    return <FormattedMessage id={`ui-users.settings.limits.${type}.error.message`} />;
+    return (
+      <SafeHTMLMessage
+        id="ui-users.settings.limits.validation.message"
+        values={{ min, max }}
+      />
```

Comment:
Is there a reason we need `SafeHTMLMessage` here? The translation value shows no HTML: 
```
  "settings.limits.validation.message": "Must be blank or a number from {min} to {max}",
```
NB: technically, there should be a [`number` type](https://formatjs.io/docs/icu-syntax#number-type) given to the message formatter: 
```
  "settings.limits.validation.message": "Must be blank or a number from {min, number} to {max, number}",

```